docs/running-porch-in-kind-cluster.md

```diff
@@ -0,0 +1,253 @@
+    --namespace=porch-test \
+    --type=kubernetes.io/basic-auth \
+    --from-literal=username=nephio \
+    --from-literal=password=secret
+```
+
+Now, we define the test Repository CR in Porch:
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Repository
+
+metadata:
+  name: testrepo
+  namespace: porch-test
+
+spec:
+  description: testrepo
+  content: Package
+  deployment: false
+  type: git
+  git:
+    repo: http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git
+    directory: /
+    branch: main
+    secretRef:
+      name: gitea-porch-secret
+EOF
+```
+
+Check that the Repository has been correctly created:
+
+```
+kubectl get repositories -n porch-test
+```
+
+Sample output:
+```
+NAME       TYPE   CONTENT   DEPLOYMENT   READY   ADDRESS
+testrepo   git    Package   false        True    http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git
+```
+Check the logs of the porch-server
+
+```
+kubectl logs deployment/porch-server -n porch-system
+```
+
+Sample output:
+
+```
+...
+
+I0124 11:23:10.400567       1 background.go:134] Repository added: porch-test:testrepo
+I0124 11:23:10.533213       1 repository.go:364] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: poll started
+I0124 11:23:10.541438       1 background.go:137] Repository modified: porch-test:testrepo
+I0124 11:23:10.681949       1 git.go:1702] discovered 0 packages @ecf8bed2663745165615f9390e5e3df7c0196d93
+I0124 11:23:10.681964       1 repository.go:528] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: addSent 0, modSent 0, delSent for 0 old and 0 new repo packages
+I0124 11:23:10.681968       1 repository.go:397] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: refresh finished in 0.099783 secs
+I0124 11:23:10.681975       1 repository.go:365] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: poll finished in 0.148768 secs
+I0124 11:23:10.681979       1 repository.go:364] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: poll started
+I0124 11:23:10.779443       1 repository.go:397] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: refresh finished in 0.047841 secs
+I0124 11:23:10.779453       1 repository.go:365] repo git://http://gitea-http.gitea.svc.cluster.local:3000/nephio/testrepo.git/: poll finished in 0.097474 secs
+...
+```
+
+
+## Teardown the custom deployments
+
+Destroy the gitea kpt pkg resources:
+
+```
+kpt live destroy .build/deploy/kpt_pkgs/gitea-dev
+```
+
+Alternatively, we can use kubectl:
+
+```
+kubectl delete --wait --recursive --filename ./.build/deploy/kpt_pkgs/gitea-dev/
+```
+
+Destroy the porch kpt pkg resources:
+
+```
+kpt live destroy .build/deploy/kpt_pkgs/porch
+```
+
+Again, alternatively, we can use kubectl:
+
+```
+kubectl delete --wait --recursive --filename ./.build/deploy/porch/
+```
```

Comment:
You could add the command to delete the kind cluster just for completeness

```
kind delete cluster -n dev
```

Reply:
done